### PR TITLE
Prometheus packages

### PR DESCRIPTION
**Pull request by celskeggs**
These add packages for prometheus and the prometheus node exporter.

Part of #180.

A review here should include independent verification of the hashes in SHA512SUM.UPSTREAM for all upstream code. See the [with-prometheus](https://github.com/sipb/homeworld-upstream/tree/with-prometheus) branch on homeworld-upstream for my copy of the source packages. (Which you should be verifying, not trusting.)

### CODE REVIEW

File: building/build-debs/homeworld-prometheus-node-exporter/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+go-bin-*
+prometheus-*.tar.xz
+promu-*.tar.xz
+gopath
+./bin/
```
Comment (andersk) [inline building/build-debs/homeworld-prometheus-node-exporter/.gitignore]: .gitignore doesn’t work that way—you mean `/bin/` (and you possibly mean to have a leading `/` on most .gitignore entries in the whole repository?).

File: building/build-debs/homeworld-prometheus-node-exporter/debian/control [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Source: homeworld-prometheus-node-exporter
+Section: misc
+Priority: extra
+Maintainer: Cel Skeggs <cela@mit.edu>
+Build-Depends: debhelper (>= 9), rsync
```
Comment (celskeggs) [inline building/build-debs/homeworld-prometheus-node-exporter/debian/control]: This isn't supposed to include rsync.

File: building/build-debs/homeworld-prometheus/debian/control [hunk 3]
```diff
@@ -0,0 +1,14 @@
+Source: homeworld-prometheus
+Section: misc
+Priority: extra
+Maintainer: Cel Skeggs <cela@mit.edu>
+Build-Depends: debhelper (>= 9), rsync
```
Comment (celskeggs) [inline building/build-debs/homeworld-prometheus/debian/control]: Also not supposed to include rsync.

**Comment by ichung** [comment]
Can't verify `sipb-promu`; I find

    af7cdf5fec21f1a90ab2e608470f3bdd85fc03b57c99bf90dc86efcf3831cc9422604ea1acc26b23cfef0e7b258cf45786097bb9e63259734ef26f89a6417838  promu-sipb-0.1.1.tar.xz

**Comment by celskeggs** [comment]
Fixed. I'd used the wrong prefix on my git archive export.

**Pull request merged**